src/workspace/src/home/index.ts

```diff
@@ -132,16 +140,16 @@ export async function registerHome(): Promise<void> {
         await handleAppSelection(result.data)
       }
     } else {
-      console.warn('Unable to execute result without data being passed')
+      console.warn("Unable to execute result without data being passed")
     }
   }
 
   const cliProvider: CLIProvider = {
-    title: 'Adaptive Workspace',
+    title: "Adaptive Workspace",
     id: PROVIDER_ID,
-    icon: `${BASE_URL}/images/icons/adaptive.png`,
+    icon: ADAPTIVE_LOGO,
```

Comment:
no effective change here, ofc, but do you know which part of the WS UI this affects?